modules/planning/tasks/optimizers/open_space_trajectory_generation/open_space_trajectory_provider.cc

```diff
@@ -125,6 +125,11 @@ Status OpenSpaceTrajectoryProvider::Process() {
 
     if (is_generation_thread_stop_) {
       GenerateStopTrajectory(trajectory_data);
+      double standstill_acceleration =
+          vehicle_state.linear_velocity() >= -0.00001 ? -0.3 : 0.3;
```

Comment:
make +/- 0.3 this a configurable parameter.

Reply:
Fixed. Thanks